src/parser/template.rs

```diff
@@ -20,7 +20,7 @@ impl Parser for Template {
         let reader = std::io::BufReader::new(file);
 
         for (count, line) in reader.lines().enumerate() {
-            let line = line?;
+            let _ = line?;
```

Comment:
Could you use "_line" instead of "_"? Although "line" is an unused variable, I think it's better to avoid using just an underscore for readability as a module template.

Reply:
Sure, here it is